pkg/config/config.go

```diff
@@ -269,6 +288,23 @@ func NewConfig(confString string, strictMode bool, c *cli.Context, baseFlags []c
 				Enabled:    true,
 				AllowPause: false,
 				ProbeMode:  CongestionControlProbeModePadding,
+				ProbeConfig: CongestionControlProbeConfig{
+					BaseInterval:  3 * time.Second,
```

Comment:
This used to be 5 seconds, but reducing to 3 seconds as we are doing shorter (hopefully less disruptive) probes.

Reply:
great to see these as config options!